dev/source/docs/ros2-gazebo.rst

```diff
@@ -35,15 +35,29 @@ It's recommended to set this in your `~/.bashrc` file.
 
     export GZ_VERSION=harmonic
 
-Update ROS dependencies:
+Add Gazebo APT sources.
+
+.. code-block:: bash
+
+  sudo apt install wget
+  wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg
+  echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null
+  sudo apt update
+
+Add Gazebo sources to `rosdep`.
+
+.. code-block:: bash
+
+  wget https://raw.githubusercontent.com/osrf/osrf-rosdep/master/gz/00-gazebo.list -O /etc/ros/rosdep/sources.list.d/00-gazebo.list
+  rosdep update
+
+Update ROS and Gazebo dependencies:
 
 .. code-block:: bash
 
     cd ~/ardu_ws
     source /opt/ros/humble/setup.bash
-    sudo apt update
-    rosdep update
-    rosdep install --from-paths src --ignore-src -r
+    rosdep install --from-paths src --ignore-src -y
```

Comment:
You should leave in `sudo apt update` and `rosdep update`. It's necessary. Please only remove the `-r` flag.